### DISCUSSION TITLE
Custom collection names?

### DISCUSSION THREAD

**Discussion by rijkdw** [Josuto/node-abstract-repository#230] [Ideas]
Good day,

First and foremost, thank you for this excellent project.  Has the possibility to let the implementation of `MongooseRepository` decide on the name of the underlying collection it will use, been explored before?  I'd be happy to assist should you approve the idea.

Thanks again for monguito!

Best wishes,
Rijk

**Comment by Josuto** [comment]
Hi @rijkdw! Many thanks for your kind words, and I'm happy you are enjoying Monguito! 🥳

I'd already thought about the possibility of enabling developers to pick custom names for collections but unfortunately had no time to do so myself. Therefore, please be my guest to come up with a solution 😃

I will try to create an issue from this discussion tomorrow so that we can discuss a solution proposal and later relate it to the actual implementation, if that's ok by you 😉

Many thanks again!

**Reply by Josuto** [reply]
Morning! Created the issue #233 to discuss a solution proposal.
